anet-dictionary.yml

```
@@ -71,6 +71,7 @@ siteClassification: demo
 fields:
 
   task:
+    allTasksLabel: Top-level Objectives
```

```suggestion
    allTasksLabel: Objectives
```

Inconsistent with _Top-level Organizations_ label, I know, feel free to change that as well.

This is just more sensible default in my opinion.